src/components/ProjectTile/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useContext } from "react";
+import {
+  Description,
+  EachSlide,
+  ProjectContainer,
+  SlideContainer,
+  Text,
+  Title,
+} from "./style";
+import { Slide } from "react-slideshow-image";
+import "react-slideshow-image/dist/styles.css";
+import { ModalContext } from "../../context/ModalContextProvider";
+import FirstProjectModalContent from "./FirstProjectModalContent";
+import { SmallButton } from "../../styles";
+
+interface SlideShowProps {
+  images: string[];
+}
+
+interface DescriptionData {
+  title: string;
+  text: string;
+  repo_link: string;
+  design_link: string;
```

Comment:
props should be in camelCase:
```suggestion
  designLink: string;
```